### PR TITLE
update mp-jwt tck tests to repeat with cdi 2.0

### PR DESCRIPTION
**Pull request by brutif**

### CODE REVIEW

**Comment by brutif** [comment]
these include Tom Evan's bnd changes from pull 1989. 
#build

**Comment by brutif** [comment]
pb is good

File: dev/com.ibm.ws.security.mp.jwt_fat_tck/.classpath [hunk 1]
```diff
@@ -4,5 +4,13 @@
 	<classpathentry kind="src" path="fat/resources"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="lib" path="/build.sharedResources/lib/junit/old/junit.jar">
+		<attributes>
+			<attribute name="bsn" value="C:\Users\IBM_ADMIN\libertyGit\open-liberty\dev/build.sharedResources/lib/junit/old/junit.jar"/>
```
Comment (ayoho) [inline dev/com.ibm.ws.security.mp.jwt_fat_tck/.classpath]: Don't think that's gonna work. If the personal build didn't fail, is this needed?

File: dev/com.ibm.ws.security.mp.jwt_fat_tck/fat/resources/permissions.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    version="7">
+   
+   <permission>
+       <class-name>java.lang.RuntimePermission</class-name>
+       <name>getClassLoader</name>
+   </permission> 
+       
+   <permission>
+       <class-name>javax.security.auth.PrivateCredentialPermission</class-name>
+       <name>* * "*"</name>
```
Comment (ayoho) [inline dev/com.ibm.ws.security.mp.jwt_fat_tck/fat/resources/permissions.xml]: Is this a valid permissions name? I would've expected just `<name>*</name>`.

Comment (brutif) [reply dev/com.ibm.ws.security.mp.jwt_fat_tck/fat/resources/permissions.xml]: yep that strange syntax is actually required.  Copied from security manager tests.

Comment (brutif) [reply dev/com.ibm.ws.security.mp.jwt_fat_tck/fat/resources/permissions.xml]: actually I'm going to cancel this request and replace it with another that doesn't have a merge conflict

**Comment by brutif** [comment]
replacing with #2127

**Pull request closed without merging**